pyproject.toml

```diff
@@ -49,8 +49,7 @@ dependencies = [
     "rich>=13.0.0",  # databinder
     "aiofile",  # compatible versions controlled through miniopy-async
     "make-it-sync",  # compatible versions controlled through func_adl
-    "ruamel.yaml>=0.18,<0.18.7",  # FIXME: ruamel-yaml v0.18.7 has a breaking API change relative to v0.18.6
-    "ccorp-yaml-include-relative-path>=0.0.4",
+    "ruamel.yaml>=0.18.7",
```

Comment:
@BenGalewsky If you can wait to release a `v3.1.0` until this PR is merged that would be great so that the upper bounds on `ruamel.yaml` that I added in PR #542 never make it into a release. :+1: